configs/opencv-ovms/instance_segmentation_demo/python/common/python/openvino/model_zoo/model_api/models/__init__.py

```diff
@@ -0,0 +1,79 @@
+"""
+ Copyright (C) 2021-2023 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+
+from .bert import BertEmbedding, BertNamedEntityRecognition, BertQuestionAnswering
+from .background_matting import ImageMattingWithBackground, VideoBackgroundMatting, PortraitBackgroundMatting
+from .centernet import CenterNet
+from .classification import Classification
+from .deblurring import Deblurring
+from .detection_model import DetectionModel
+from .detr import DETR
+from .ctpn import CTPN
+from .faceboxes import FaceBoxes
+from .hpe_associative_embedding import HpeAssociativeEmbedding
+from .image_model import ImageModel
+from .instance_segmentation import MaskRCNNModel, YolactModel
+from .model import Model
+from .monodepth import MonoDepthModel
+from .nanodet import NanoDet, NanoDetPlus
+from .open_pose import OpenPose
+from .retinaface import RetinaFace, RetinaFacePyTorch
+from .segmentation import SegmentationModel, SalientObjectDetectionModel
+from .ssd import SSD
+from .ultra_lightweight_face_detection import UltraLightweightFaceDetection
+from .utils import DetectionWithLandmarks, InputTransform, OutputTransform, RESIZE_TYPES
+from .yolo import YOLO, YoloV3ONNX, YoloV4, YOLOF, YOLOX
+
+__all__ = [
```

Comment:
Don't need to change it now but it would be good to see if the demo still works if we remove all of the unused models from the list.

Reply:
I can refactor it in a different PR